canonical-web-and-design/snapcraft.io
# Refactor publisher views

## Done

Split the publisher views into different files for the features that we offer when managing a Snap.

## Issue / Card

Fixes #2519

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure everything work when editing a Snap.
